src/test/java/com/google/api/codegen/testdata/ruby_doc_message_library.baseline

```diff
@@ -463,7 +495,34 @@ end
 # limitations under the License.
 
 module Google
+  ##
+  # The `Google::Protobuf` module provides the following types:
+  #
+  # Class | Description
+  # ----- | -----------
+  # Google::Protobuf::ListValue | +ListValue+ is a wrapper around a repeated field of values.
+  # Google::Protobuf::NullValue | +NullValue+ is a singleton enumeration to represent the null value for the
+  # Google::Protobuf::Struct | +Struct+ represents a structured data value, consisting of fields
+  # Google::Protobuf::Value | +Value+ represents a dynamically typed value which can be either
+  #
   module Protobuf
+    # +ListValue+ is a wrapper around a repeated field of values.
```

Comment:
I moved everything so they're in alphabetical order now. Easier than maintaining a sorted and unsorted list.